oxide/crates/core/tests/auto_content.rs

```diff
@@ -297,9 +297,14 @@ mod auto_content {
             ("foo.jpg", Some("xl:font-bold")),
             // A file that is ignored
             ("foo.html", Some("lg:font-bold")),
+            // A svelte file with `class:foo="bar"` syntax
+            ("index.svelte", Some("<div class:px-4='condition'></div>")),
```

Comment:
In Svelte, expression would be in squiggly brackets and not quotes
`<div class:px-4={condition}>`